### PR TITLE
Embedded cwltool/salad via heterodon.

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

File: src/ci/bin/test.inc.sh [hunk 1]
```diff
@@ -312,14 +311,11 @@ cromwell::private::pull_common_docker_images() {
     docker pull ubuntu
 }
 
-cromwell::private::install_cwltool() {
+cromwell::private::install_cwltest() {
     # TODO: No clue why these are needed for cwltool. If you know please update this comment.
```
Comment (mcovarr) [inline src/ci/bin/test.inc.sh]: is this the time to update this comment?

**Comment by mcovarr** [comment]
and another :+1: for the `heterodon` name

File: src/ci/bin/test.inc.sh (hunk 1 above)
Comment (kshakir) [reply src/ci/bin/test.inc.sh]: I still don't know why this is required. 🤔 So I'm leaving it until someone gives an answer.

File: cwl/src/main/scala/cwl/CwltoolRunner.scala [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+sealed trait CwltoolRunner {
+  def salad(file: BFile): String
+}
+
+object CwltoolRunner {
+  private lazy val config = ConfigFactory.load
+
+  lazy val instance: CwltoolRunner = {
+    val runnerClass = config.getString("cwltool-runner.class")
+    Class.forName(runnerClass).newInstance().asInstanceOf[CwltoolRunner]
```
Comment (danbills) [inline cwl/src/main/scala/cwl/CwltoolRunner.scala]: ToL Could just import this lib/class and `new` it up, no need for jvm-level instantiation IMO

**Pull request merged**